environments/eck-ror/start.sh

```diff
@@ -90,12 +90,38 @@ if [[ -z $ES_VERSION || -z $KBN_VERSION ]]; then
   show_help
 fi
 
+DOCKERFILE_DIR="./images/ubuntu"
```

Comment:
in both environments, we use the same image. Maybe we should create a `common` folder at `./environments/` folder level and place them common images definitions (like the node apm app)